api/v1/ptpoperatorconfig_types.go

```diff
@@ -58,6 +65,24 @@ type PtpOperatorConfigList struct {
 	Items           []PtpOperatorConfig `json:"items"`
 }
 
+// PtpEventConfig defines the desired state of event framework
+type PtpEventConfig struct {
```

Comment:
is it possible to have in one cluster different thresholds for different nodes?

if that is the case I think we can have here just a flag to enable disable the event bus sidecar and move the thresholds as part of the ptp-config so it will be applied with the policy.

In general I think the thresholds depend on the policy you use so they need to be there

Reply:
@SchSeba  it's possible, not likely. But there is a lot of overhead of providing this functionality at this point., i.e. moving this to ptp-config.
if I move this to ptp-config then sidecar needs to watch for ptp-config changes in order to know these values before it can process logs coming from Linux PTP-daemon. That means either the Linux ptp daemon has to send me these values or I need to read the .conf files periodically.

Reply:
I think 4.9 is primarily SNO, I can do this in the next release and capture this feedback in the user story?

Reply:
The problem is the API is v1 this means we will need to support this for a long time before we are able to depricate.

we already pass all the information to the linuxptp using a configmap you can do the same as the linuxptp is doing right now

Reply:
The issue is sidecar is already running, ptp config can(will)   be applied after the sidecar is in running. So that mean sidecar should be able to read configmap changes ? is that correct

Reply:
Fixed: Moved to ptpConfig